grafana/README.md

```diff
@@ -0,0 +1,25 @@
+# Grafana
+
+This directory contains useful out-of-box dashboards.
+
+# Constraints
+
+Dashboards contain [summary](../grafana/dashboards/summary) and [single](../grafana/dashboards/single) dashboards.
+* summary: Aggregated dashboards for a cluster(`alias`).
+* single: single dashboards for a single resource(e.g. `node` or `pod`).
+
+summary dashboards use `alias` as a template variable. `alias` can be set as a cluster name
```

Comment:
I think having something like this is a sensible thing to do, but `alias` is too generic, so I think `cluster` would be much more clear on what this selection is supposed to represent.